src/lib/eslint/JavascriptEslintStrategy.ts

```diff
@@ -1,13 +1,12 @@
 import { EslintStrategy } from './BaseEslintEngine';
 import {ENGINE, LANGUAGE} from '../../Constants';
-import {RuleViolation} from '../../types';
+import {ESRule, LooseObject, RuleViolation} from '../../types';
 import { Logger } from '@salesforce/core';
 import { ProcessRuleViolationType } from './EslintCommons';
+import path = require('path');
 
 const ES_CONFIG = {
-	"baseConfig": {
-		"extends": ["eslint:recommended"]
-	},
+	"baseConfig": {},
```

Comment:
Is this still required?

Reply:
I don't think so, but its presence doesn't seem to do any harm and I felt like deleting the property outright was too aggressive to do on a whim.